core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -264,6 +275,24 @@ trait WorkspaceComponent {
       } yield workspace
     }
 
+    def listWorkspaceGoogleProjectsWithIncorrectBillingAccounts(): ReadAction[Seq[(GoogleProjectId, Option[RawlsBillingAccountName])]] = {
```

Comment:
might be worth documenting / renaming to reflect that the billingAccount that is returned is the right one and not the incorrect one (if i'm understanding this correctly)